include/bluetooth/l2cap.h

```diff
@@ -26,21 +26,61 @@
 extern "C" {
 #endif
 
-/** L2CAP header size, used for buffer size calculations */
+/** L2CAP PDU header size, used for buffer size calculations */
 #define BT_L2CAP_HDR_SIZE               4
 
+/** Maximum Transmission Unit (MTU) for an outgoing L2CAP PDU. */
```

Comment:
May be we can align terminology with the Bluetooth spec as well? Basically, max size of PDU called MPS in the spec. MTU is max size of SDU.

Reply:
That is only used for CoC though, for fixed channels/basic mode it seems MTU is used and sometimes even in case of CoC e.g:

```
When using an L2CAP channel with a dynamically allocated CID, the
ATT_MTU shall be set to the L2CAP MTU size.
```